js/game.js

```diff
@@ -56,13 +58,26 @@ function draw() {
  		}
 		drawText("garbed in: " + garbed_in, 20, false, 100, "yellow");
 
-		for(let i=0; i<player.spells.length; i++){
-			let spellText = (i+1) + ") " + (player.spells[i] || "");
-			drawText(spellText, 20, false, 120+i*40, "aqua");
-		}
-	}
+    drawSpells()
 }
 
+
+function drawSpells() {
+  
+  for(let i=0; i<player.spells.length; i++){
```

Comment:
How about a `forEach` here since player.spells is an array?

Reply:
Sure, I forgot about `foreach` kkk